systemtest/src/test/java/io/strimzi/systemtest/TopicST.java

```diff
@@ -181,6 +195,45 @@ void testBigAmountOfTopicsCreatingViaKafka() {
         }
     }
 
+    @Tag(NODEPORT_SUPPORTED)
+    @Tag(SCALABILITY)
+    @Test
+    void testBigAmountOfTopicsCreatingViaAdminClient() throws ExecutionException, InterruptedException, TimeoutException {
+
+        KafkaResource.kafkaEphemeral(CLUSTER_NAME, 3, 1)
+            .editSpec()
+                .editKafka()
+                    .editListeners()
+                        .withNewKafkaListenerExternalNodePort()
+                            .withTls(false)
+                        .endKafkaListenerExternalNodePort()
+                    .endListeners()
+                .endKafka()
+            .endSpec()
+            .done();
+
+        Properties properties = new Properties();
+        properties.setProperty(AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, AbstractKafkaClient.getExternalBootstrapConnect(NAMESPACE, CLUSTER_NAME));
```

Comment:
why the admin client goes through an external listener?

Reply:
The client runs locally, or?

Reply:
The client runs locally.